fetch-json/index.html

```diff
@@ -20,31 +17,30 @@ <h1>Fetch json example</h1>
 
   </body>
   <script>
-    var myList = document.querySelector('ul');
+    const myList = document.querySelector('ul');
 
     fetch('products.json')
-    .then(function(response) {
+    .then((response) => {
       if (!response.ok) {
-        throw new Error("HTTP error, status = " + response.status);
+        throw new Error(`HTTP error, status = ${response.status}`);
       }
       return response.json();
     })
     .then(function(data) {
-      for(var i = 0; i < data.products.length; i++) {
-        var listItem = document.createElement('li');
-        listItem.innerHTML = '<strong>' + data.products[i].Name + '</strong>';
-        listItem.innerHTML +=' can be found in ' + data.products[i].Location + '.';
-        listItem.innerHTML +=' Cost: <strong>£' + data.products[i].Price + '</strong>';
+      for(const product of data.products) {
+        const listItem = document.createElement('li');
+        listItem.innerHTML  = `<strong> ${product.Name} </strong>`;
+        listItem.innerHTML += ` can be found in ${product.Location}.`;
+        listItem.innerHTML += ` Cost: <strong>£${data.product.Price}</strong>`;
```

Comment:
Merge into one template literal? (No strong feelings but I don't like concatenating multiple strings just for the purpose of line widths.)

Reply:
Done. But I will get rid of the `innerHTML` here, too (in another commit): this is a security risk as `product.Location` and `product.Price` both are from an external file.